cmd/status.go

```diff
@@ -12,19 +12,9 @@ import (
 )
 
 var (
-	override []string
-	names    status.CustomName
-	output   = utils.EnumFlag{
-		Allowed: []string{
-			status.OutputEnv,
-			status.OutputJson,
-			status.OutputPretty,
-			status.OutputToml,
-			status.OutputYaml,
-		},
-		Value: status.OutputPretty,
-	}
-
+	override  []string
+	names     status.CustomName
+	output    = utils.OutputFlag(true /* allow env output */)
```

Comment:
```suggestion
	override []string
	names    status.CustomName
	output   = utils.EnumFlag{
		Allowed: append([]string{status.OutputEnv, utils.OutputDefaultAllowed}),
		Value: status.OutputPretty,
	}
```

Can we do something like this ^? I think it reads more explicit than wrapping a function.